packages/contracts/deploy/20_permissions/20_plugin-registrty-permissions.ts

```diff
@@ -28,21 +28,23 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     hre
   );
 
-  const REGISTER_PLUGIN_REPO_PERMISSION_ID = ethers.utils.id(
-    'REGISTER_PLUGIN_REPO_PERMISSION'
-  );
-
-  // Gransting Permissions
-  grantTx = await managingDaoContract.grant(
-    pluginRepoRegistryAddress,
-    pluginRepoFactoryAddress,
-    REGISTER_PLUGIN_REPO_PERMISSION_ID
-  );
-  await grantTx.wait();
-
-  console.log(
-    `Granted the REGISTER_PLUGIN_REPO_PERMISSION of 'pluginRepoRegistry' (${pluginRepoRegistryAddress}) to 'pluginRepoFactory' (${pluginRepoFactoryAddress}) (tx: ${grantTx.hash})`
-  );
+  // Grant `REGISTER_PLUGIN_REPO_PERMISSION` of `PluginRepoRegistry` to `DAOFactory`.
+  // Grant `UPGRADE_REGISTRY_PERMISSION` of `PluginRepoRegistry` to `ManagingDAO`.
+  const revokePermissions = [
+    {
+      operation: Operation.Grant,
+      where: {name: 'PluginRepoRegistry', address: pluginRepoRegistryAddress},
+      who: {name: 'PluginRepoFactory', address: pluginRepoFactoryAddress},
+      permission: 'REGISTER_PLUGIN_REPO_PERMISSION',
```

Comment:
Isn't it better to use the exact same value that the contract uses internally, instead of hardcoding a string here?   `ContractX.AAA_PERMISSION_ID()`

Reply:
`For these tests, shouldn't we be using the prefunded wallets from Hardhat?` no need
`Isn't it better to use the exact same value that the contract uses internally, instead of hardcoding a string here? ContractX.AAA_PERMISSION_ID()` didn't thought about it since, it was already like that previously,
please add it to the following task, since it is repeating alot.

Reply:
If we are deploying the release candidate version without this, then this will be the thing. It was an extra sanity check. Feel free to resolve here.